packages/server/modules/workspaces/events/eventListener.ts

```diff
@@ -486,14 +494,20 @@ export const workspaceTrackingFactory =
     getDefaultRegion,
     getWorkspacePlan,
     getWorkspaceSubscription,
-    getUserEmails
+    getUserEmails,
+    getWorkspaceModelCount,
+    getWorkspaceProjectCount,
+    getWorkspaceSeatCount
   }: {
     getWorkspace: GetWorkspace
     countWorkspaceRole: CountWorkspaceRoleWithOptionalProjectRole
     getDefaultRegion: GetDefaultRegion
     getWorkspacePlan: GetWorkspacePlan
     getWorkspaceSubscription: GetWorkspaceSubscription
     getUserEmails: FindEmailsByUserId
+    getWorkspaceModelCount: GetWorkspaceModelCount
+    getWorkspaceProjectCount: GetWorkspacesProjectsCounts
```

Comment:
misleading naming - the type is actually GetWorkspacesProjectsCounts (plural Workspaces), and yet you've named it as getWorkspaceProjectCount, leading future devs (and possibly yourself) to believe that this function only returns data for 1 workspace